website.i2pd.i2pd.appdata.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2018 Viktor Villainov <supervillain@riseup.net> -->
 <component type="desktop">
-  <id>i2pd.desktop</id>
+  <id>website.i2pd.i2pd.desktop</id>
```

Comment:
No need for `.desktop` here.

Reply:
ok, just took signal manifest as example. will fix it